scripts/ci/install-minikube.sh

```diff
@@ -1,98 +1,122 @@
 #!/bin/bash
 
-set -v
-
 OS=linux
 ARCH=amd64
-TARGET_DIR="/usr/bin"
+TARGET_DIR=/usr/bin
 
-MINIKUBE_VERSION="v0.24.1"
+MINIKUBE_VERSION="v0.25.2"
 MINIKUBE_URL="https://github.com/kubernetes/minikube/releases/download/$MINIKUBE_VERSION/minikube-$OS-$ARCH"
 
 KUBECTL_VERSION="v1.9.0"
 KUBECTL_URL="https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/$OS/$ARCH/kubectl"
 
-[ -z "$MINIKUBE_FLAGS" ] && MINIKUBE_FLAGS="--extra-config=kubelet.CgroupDriver=systemd"
-[ -z "$MINIKUBE_VM_DRIVER" ] && MINIKUBE_VM_DRIVER=none
+export MINIKUBE_WANTUPDATENOTIFICATION=false
+export MINIKUBE_WANTREPORTERRORPROMPT=false
+export MINIKUBE_HOME=$HOME
+export CHANGE_MINIKUBE_NONE_USER=true
+export KUBECONFIG=$HOME/.kube/config
 
 uninstall_binary() {
-	local prog=$1
-	sudo rm -f ${TARGET_DIR}/$prog
+        local prog=$1
+        sudo rm -f $TARGET_DIR/$prog
 }
 
 install_binary() {
-	local prog=$1
-	local url=$2
-	local target=${TARGET_DIR}/$prog
-
-	[ -f $target ] && return
+        local prog=$1
+        local url=$2
 
-	wget --no-check-certificate -O $prog $url
-	if [ $? != 0 ]; then
-		echo "failed to download $url"
-		exit 1
-	fi
+        wget --no-check-certificate -O $prog $url
+        if [ $? != 0 ]; then
+                echo "failed to download $url"
+                exit 1
+        fi
 
-	chmod a+x $prog
-	sudo mv -f $prog $target
+        chmod 0777 $prog
```

Comment:
why 777 ? a+x : executable for everyone is enough and more appropriate

Reply:
done